src/server/sproutVideo/video.js

```diff
@@ -22,4 +42,10 @@ class Video {
   }
 }
 
+function getContentLength(formData) {
+  return new Promise((resolve, reject) => {
+    formData.getLength((err, length) => err ? reject(err) : resolve(length));
+  });
```

Comment:
Can we achieve this with `util.promisify`?

Reply:
It doesn't work. `TypeError: Cannot read property 'bind' of undefined` error is thrown